server/sonar-server/src/main/java/org/sonar/ce/log/CeProcessLogging.java

```diff
@@ -28,7 +29,7 @@
 public class CeProcessLogging extends ServerProcessLogging {
 
   public CeProcessLogging() {
-    super("ce", "%X{ceTaskUuid}");
+    super(LogbackHelper.LogProcess.CE, "%X{ceTaskUuid}");
```

Comment:
it's even better after the merge : using existing ProcessId is far better than defining new enum LogProcess.